clang/test/dpct/cmake_migation/case_009/expected.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.24)
```

Comment:
could you try to merge the expected.txt and input.cmake,  to make it easy to check the migration result.

Reply:
If each line in  input.cmake is checked by using "// CHECK:...", the checkline format like "// CHECK:..." in inpute.cmake will be treated as cmake script code.  So I suggest using current check way.